libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/api/MapboxAudioGuidance.kt

```diff
@@ -209,7 +209,7 @@ internal constructor(
          * Get the registered instance or create one and register it to [MapboxNavigationApp].
          */
         @JvmStatic
-        fun getInstance(): MapboxAudioGuidance = MapboxNavigationApp
+        fun getRegisteredInstance(): MapboxAudioGuidance = MapboxNavigationApp
```

Comment:
If this was a stable api already. The solution is to add a `@Deprecated` annotation. I'm choosing not to go that route right now, because the release is in beta and these types of issues are expected